sigstore/_cli.py

```diff
@@ -143,13 +143,18 @@ def _verify(files, certificate_path, signature_path, cert_email):
     logger.debug(f"Using signature from: {signature_path.name}")
     signature = signature_path.read()
 
+    verified = True
     for file in files:
-        click.echo(
-            verify(
-                file=file,
-                certificate=certificate,
-                signature=signature,
-                cert_email=cert_email,
-                output=click.echo,
-            )
-        )
+        if verify(
+            file=file,
+            certificate=certificate,
+            signature=signature,
+            cert_email=cert_email,
+        ):
+            click.echo(f"OK: {file.name}")
+        else:
+            click.echo(f"FAIL: {file.name}")
+            verified = False
```

Comment:
N.B.: `cosign` doesn't seem to have a standard non-JSON output for verification yet, so I just made something up here. Longer term, we'll probably want some sort of `VerificationResult` model that's either truthy or falsey that we can display for each step.

Reply:
I think that's fine creating your own, you don't need to replicate cosign, its possible you might think of a better way of implementation.

Reply:
Generally our philosophy here is to be as much of a drop-in replacement for the parts of cosign that we care about as possible. But I agree, where cosign has no precedent we can do what's best for us, but we should also make sure to make an issue to 'upstream' this feature so we don't diverge significantly.

Reply:
Makes sense. Given that, I think I'll spend some more time on this PR getting the `VerificationResult` in place. Thinking more about it, the signature should really be `Optional[VerificationResult]`, since verification is all-or-none.

Reply:
Agreed!